libpod/runtime.go

```diff
@@ -211,6 +213,11 @@ func (r *Runtime) Shutdown(force bool) error {
 
 	r.valid = false
 
+	for _, mount := range r.Mounts {
```

Comment:
I don't know if I like this. Should we store the mounts with the container instead, and have RemoveContainer do the unmounting? We need to remove all containers when doing a force shutdown anyways.

Reply:
Moving this to containwres is fine, but I need to know when the container is finished running.